woocommerce/woocommerce-android
# Free Trial: Handle non jetpack sites in More menu header

Summary
==========
Fix issue #8851 by updating the More Menu header to hide the drop down button and keeping the color scheme. 

Since the header is also a button that opens the store switch, it can be disabled through the [MoreMenuViewState.isStoreSwitcherEnabled] flag (which already covers the non-jetpack scenario).

But considering the button is also a header with store information, we want to just hide the drop down arrow to reflect the disabled appearance. The custom color scheme introduced in this PR ensures that.

Screenshots
==========
| Light Mode  | Dark Mode |
| ------------- | ------------- |
| ![Screenshot_20230420_211126](https://user-images.githubusercontent.com/5920403/233512079-0bb0bf7f-ba6d-4ac4-8e23-ea475879df91.png) | ![Screenshot_20230420_211136](https://user-images.githubusercontent.com/5920403/233512090-576bfcd8-3f22-40d9-ab91-e3cfa43bd9b5.png) |

How to Test
==========
1. Log in the app using a store not connected to Jetpack
2. Go to the More menu section of the app and verify that the Header is not clickable, the profile picture shows a placeholder and the drop down arrow is hidden.

Update release notes:
